src/shared/Consts.js

```diff
@@ -0,0 +1,8 @@
+export const FLEX = {
+  '1_3': '0 0 520px',
```

Comment:
What does `1_3` mean? Also, using constants for things like `1_3` will make it easier to later on access these properties, it will reduce the common typo mistakes, and it will make it easier to refactor if you ever decide to rename `1_3` to something different.